test/validation/org/apache/cassandra/HarnessTest.java

```diff
@@ -180,7 +180,7 @@ static URL getConfigURL(String yamlPath)
         }
     }
 
-    public String getTestName(String yamlPath)
+    static String getTestName(String yamlPath)
```

Comment:
Why change this to static? Not that I disagree, but you don't appear to be using it in a static context anywhere in this PR